### PR TITLE
Update Doxypress to 1.4.0

### PR DESCRIPTION
**Pull request by rleh**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,7 +5,6 @@ WORKDIR /work
 
 ADD . /work
 
-ENV SCONS_LIB_DIR="/usr/local/lib/python3.7/dist-packages/scons"
```
Comment (rleh) [inline Dockerfile:8]: This may not have worked because we have Python3.8. So drop it.

**Review by salkinium — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (salkinium) [reply Dockerfile:8]: I think this was a specific issue with Ubuntu 18.04 or with the Python 3.7 or something.

**Comment by salkinium** [comment]
(You can push when you're ready)

**Comment by rleh** [comment]
Build and tested the docker image on my local pc :heavy_check_mark: 

Does building on Dockerhub still work or do I have to `docker push modm/modm-build`?

**Comment by salkinium** [comment]
> Does building on Dockerhub still work

Yes, it's currently building.

**Pull request merged**